packages/discovery-provider/plugins/notifications/src/utils/populateDB.ts

```diff
@@ -823,6 +825,39 @@ export const createComments = async (db: Knex, comments: CreateComment[]) => {
     .into('comments')
 }
 
+type CreateCommentThread = CommentThreadRow
+
+export const createCommentThreads = async (
+  db: Knex,
+  commentThreads: CreateCommentThread[]
+) => {
+  await db.insert(commentThreads).into('comment_threads')
+}
+
+type CreateCommentMention = Pick<CommentMentionRow, 'comment_id' | 'user_id'> &
+  Partial<CommentMentionRow>
+
+export const createCommentMentions = async (
+  db: Knex,
+  commentMentions: CreateCommentMention[]
+) => {
+  await db
+    .insert(
+      commentMentions.map((mention) => ({
+        comment_id: mention.comment_id,
+        user_id: mention.user_id,
+        created_at: new Date(Date.now()),
+        updated_at: new Date(Date.now()),
```

Comment:
Curious - do we follow this pattern (set updated_at during creation), or do we leave it null until an update?

Reply:
it is set to created on initialization is what i gather from other models